xsimlab/tests/test_model.py

```diff
@@ -462,17 +462,30 @@ class Foo:
 
         @xs.runtime(args=["main_clock_values", "main_clock_dataarray"])
         def initialize(self, clock_values, clock_array):
-            self.a = clock_values
-            assert all(self.a == [0, 1, 2, 3])
+            self.a = clock_values * 2
+            assert all(self.a == [0, 2, 4, 6])
```

Comment:
nit: numpy testing functions work with any array-like object.

```suggestion
            np.testing.assert_equal(self.a, [0, 2, 4, 6])
```